### ISSUE TITLE
1-01-01: Set status of deprecated entry to "retired"

### ISSUE THREAD

**Issue by fstuerzl** [wmo-im/wmds#267]
**Branch**
[add when created]

**Summary and Purpose**
The following entry is marked as "deprecated" in the variable name. 

notation | path | name | description
---|---|---|---
309 | \Atmosphere\Wind\Wind (surface wind direction and speed, horizontal) - deprecated | Wind (surface wind direction and speed, horizontal) - deprecated | Horizontal component of wind as speed and direction.

This does not comply with the [Guidance for Removal of Terms](https://github.com/wmo-im/wmds/wiki/Guidance-for-Removal-of-Terms). According to these rules the entry should be removed from table 1-01-01.csv and appear in the codes registry with the status "retired".

**Comment by joergklausen** [comment]
Entry confirmed to be *not present* in the branch.

**Comment by amilan17** [comment]
correct. we do not want the word "deprecated" in the CSV.

**Comment by amilan17** [comment]
@joergklausen Can you confirm that this term was deprecated sometime in 2019 and replaced by two new codes [12005](http://codes.wmo.int/wmdr/ObservedVariableAtmosphere/_12005) and [12006](http://codes.wmo.int/wmdr/ObservedVariableAtmosphere/_12006). See issue #44. The label in the codes registry always included 'deprecated': [http://codes.wmo.int/wmdr/ObservedVariableAtmosphere/_309:1](http://codes.wmo.int/wmdr/ObservedVariableAtmosphere/_309:1)  

@fstuerzl  -- it looks like you accidentally re-introduced 'unknown' and 'inapplicable' -- [https://github.com/wmo-im/wmds/compare/master...issue267.](https://github.com/wmo-im/wmds/compare/master...issue267.)

**Comment by fstuerzl** [comment]
> @fstuerzl  -- it looks like you accidentally re-introduced 'unknown' and 'inapplicable' -- [master...issue267.](https://github.com/wmo-im/wmds/compare/master...issue267.)

@amilan17, thanks for  pointing that out! I've removed the two terms again.

**Comment by joergklausen** [comment]
Yes, that sounds plausible. The migration in OSCAR/Surface has not yet taken place, though.
